### DISCUSSION TITLE
Problem connecting to a CC2652 zigbee dongle in Macos

### DISCUSSION THREAD

**Discussion by mkomm** [Koenkk/zigbee2mqtt#9830] [Devices]
Hi, 
I am new to Zigbee  and I wanted to start playing with a Zigbee dongle I have recently purchased (CC2652 BLE Simplelink 2.4G Zigbee2MQTT Thread Home Assistant bought on Aliexpress), however the connection does not seem to be working. When I plug it into my Mac (10.14.6 Mojave), it creates a couple of new entries in /dev:

crw-rw-rw-   1 root     wheel            9,  20 24 Nov 18:36 tty.usbserial-1420
crwxrwxrwx   1 root     wheel            9,  21 24 Nov 18:36 cu.usbserial-1420
crw-rw-rw-   1 root     wheel            2,   0 24 Nov 18:39 tty
crwxrwxrwx   1 michael  tty             16,   3 24 Nov 18:39 ttys003
crwxrwxrwx   1 michael  tty             16,   4 24 Nov 18:45 ttys004
crw-rw-rw-   1 root     wheel           20,  16 24 Nov 18:46 io8logtemp
crw-rw-rw-   1 root     wheel            3,   2 24 Nov 18:46 null
crw-rw-rw-   1 root     tty             15,   5 24 Nov 18:46 ptmx
crwxrwxrwx   1 michael  tty             16,   1 24 Nov 18:46 ttys001

but none of them seem to work with Zigbee2mqtt:

docker run --device=/dev/ptmx -v $(pwd)/data:/app/data -v /run/udev:/run/udev:ro -e TZ=Europe/Amsterdam koenkk/zigbee2mqtt
Using '/app/data' as data directory
Zigbee2MQTT:info  2021-11-24 18:46:53: Logging to console and directory: '/app/data/log/2021-11-24.18-46-53' filename: log.txt
Zigbee2MQTT:info  2021-11-24 18:46:53: Starting Zigbee2MQTT version 1.22.0 (commit #0a1fccf
)
Zigbee2MQTT:info  2021-11-24 18:46:53: Starting zigbee-herdsman (0.13.164)
Zigbee2MQTT:error 2021-11-24 18:46:53: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2021-11-24 18:46:53: Failed to start zigbee
Zigbee2MQTT:error 2021-11-24 18:46:53: Check https://www.zigbee2mqtt.io/information/FAQ.html#help-zigbee2mqtt-fails-to-start for possible solutions
Zigbee2MQTT:error 2021-11-24 18:46:53: Exiting...
Zigbee2MQTT:error 2021-11-24 18:46:53: Error: Error while opening serialport 'Error: Error: No such file or directory, cannot open /dev/cu.usbserial-1420'

I am puzzled by the fact that it cannot open it, since I have changed the access rights already. Any hints on how to fix it?

**Comment by francisp2** [comment]
Did you flash the stick with coordinator firmware ?

**Reply by mkomm** [reply]
I did not flash it myself but manufacturer states it has coordinater firmware flashed

**Comment by io53** [comment]
Docker on mac does not seem to support mapping devices. Have a look here https://github.com/docker/for-mac/issues/900

**Reply by mkomm** [reply]
OK, is there any other way how to run zigbee2mqtt on Macos?

**Comment by francisp2** [comment]
Do a 'bare metal' installation using npm

**Reply by mkomm** [reply]
OK, I did that, actually it makes my life quite easier. However, the problem persists. I suspect it must be some problem of compatibility of the usb stick and macos but I am unable to find a solution :(

**Comment by davebra** [comment]
Most likely MacOS requires the driver for the dongle's serial chip.
I had the same issue, my dongle is a Sonoff Zigbee 3.0 USB Dongle Plus, which uses a Silabs CP2102N UART-to-USB chip. So I simply installed the correct drivers https://www.silabs.com/developers/usb-to-uart-bridge-vcp-drivers and reconnect the usb dongle, the new interface is:
`/dev/cu.SLAB_USBtoUART`

**Reply by JulianPinzaru** [reply]
Sadly - I got the same Sonoff Zigbee 3.0 USB and it didn't work with their drivers. Thank you for a tip though
